spacy/tokens/doc.pyx

```diff
@@ -934,6 +948,20 @@ cdef class Doc:
             retokenizer.merge(self[start:end], attrs=attributes)
         return self[start]
 
+    def remove_label_if_necessary(self, attributes):
+        # More deprecated attribute handling =/
+        if 'label' in attributes:
+            attributes['ent_type'] = attributes.pop('label')
+
+    def fix_attributes(self, attributes):
+        if 'label' in attributes and 'ent_type' not in attributes:
+            if isinstance(attributes['label'], int):
+                attributes[ENT_TYPE] = attributes['label']
+            else:
+                attributes[ENT_TYPE] = self.vocab.strings[attributes['label']]
+        if 'ent_type' in attributes:
+            attributes[ENT_TYPE] = attributes['ent_type']
+
```

Comment:
`remove_label_if_necessary` and `fix_attributes` should probably be functions, not methods. At the very least, they would need to be private methods --- as we don't want to support these indefinitely going forward.

Reply:
Thank you so much for all your feedback! I'm on these changes.